charts/unified-chart/test.values.yaml

```diff
@@ -10,10 +10,11 @@ linkerd:
   enabled: true
 
 service:
-  name: access-port
-  targetPort: 3003
-  labels:
-    monitoring-metrics: enabled
+  ports:
+    - name: access-port
+      targetPort: 3003
+      labels:
+        monitoring-metrics: enabled
```

Comment:
i think this label already injected in svc - but hardcoded or something
should scrape every service? or perhaps leave it to the values.default.yaml